.github/l10n/check_l10n_issues.py

```diff
@@ -8,11 +8,121 @@
 import xml.etree.ElementTree as etree
 import json
 import os
+import subprocess
 import sys
+import tempfile
 
 script_folder = os.path.abspath(os.path.dirname(__file__))
 vpn_root_folder = os.path.realpath(os.path.join(script_folder, os.pardir, os.pardir))
 
+def fileContentsAsJSON(filepath):
+    try:
+        with open(filepath, 'r') as file:
+            content = json.load(file)
+
+            if not content:
+                print(f"No content found in: {filepath}")
```

Comment:
```suggestion
                sys.exit(f"No content found in: {filepath}")
```

You can exit directly with the message (across the file).

Reply:
I had it that way initially, but moved to `print` combined with `sys.exit(1)` to match the style in the rest of this file.